lrpar/src/lib/ctbuilder.rs

```diff
@@ -173,6 +232,27 @@ where
         self
     }
 
+    /// If set to true, `process_file_in_src` will return an error if the given grammar contains
+    /// any Shift/Reduce or Reduce/Reduce conflicts. Defaults to `true`.
+    pub fn error_on_conflicts(mut self, b: bool) -> Self {
+        self.error_on_conflicts = b;
+        self
+    }
+
+    pub fn conflicts(
```

Comment:
I'm inclined to think that this API should be explicitly marked as unstable until we're sure it's in the final form we want. There isn't an off-the-shelf way to do this, but we could add an `unstable` feature to the crate (as suggested by https://github.com/rust-lang/rfcs/issues/1491#issuecomment-345455136). Thoughts?

Reply:
I have come to the opinion that this may be too confusing, and I rather just leave a comment in the documentation that this function is unstable.

Reply:
Fair enough.